### ISSUE TITLE
Opening .eml attachements

### ISSUE THREAD

**Issue by TechDevJorge** [mimestream/new-bugs#97]
**Describe the bug**
So if I receive and email with a .eml attachment and I try to open it it will use the apple mail app to open it. I've set mimecast as the default email app and also used the Microsoft tool to set it as the default but it will still open in apple mail.

**To Reproduce**
1. Open an email with a .eml attachment
2. Open the attachment
2. That's it

**Expected behavior**
I expected the .eml attachment to just open in mimecast

**Screenshots**
Please add screenshots to help explain your problem.
Not sure if it's needed since the steps are pretty straightforward

**Configuration:**
 - macOS Big Sur
 - Mimestream Version v0.11.2]

**Comment by neiljhaveri** [comment]
Yeah, with the number of times I open .eml files every day, I would love to have this too. 

The implementation of it would be a little involved, but I'm sure I'll eventually get to it.

**Comment by TechDevJorge** [comment]
Sounds good man, so far so good though especially considering it's still in BETA.
